### DISCUSSION TITLE
Recognition Hangs Indefinitely When Completing Sentences in Nova-2-ea

### DISCUSSION THREAD

**Discussion by rudder-aniket-kale** [deepgram/community#363] [General help]
### Which Deepgram product are you using?

Deepgram API

### Details

Sometimes Deepgram **Nova-2-ea** model gets stuck in a loop where it **keeps waiting for the user to finish the sentence even after the user has actually completed the sentence**. Consequently, the user has to utter something further to somehow prompt the model to finish. 

 **Steps to reproduce**
1. Input Sentence: Start a transcription session with Deepgram Nova2.
2. Speak: Say a complete sentence or phrase clearly.
3. Wait: Observe that Deepgram Nova2 does not recognize the end of the sentence and continues waiting indefinitely.

**Expected behavior**
The model should be able to recognize when the user has completed their sentence and stop waiting for further input.

If possible, it would be helpful to investigate any previously raised issues (if any) related to this problem, and if the team can share any suggestions on how to overcome this issue on our end while you work on it, it would be of great help.

**Comment by jpvajda** [comment]
@aniket-rudder  Thanks for pointing this out and for trying out Nova-2. I'll pass this along to one of our Product Teams for evaluation.

**Comment by DamienDeepgram** [comment]
Hi @aniket-rudder,

Can you share the last few words of the transcript where this issue is happening and what url parameters you are using?
